src/test/java/teammates/test/driver/TestStringHelper.java

```diff
@@ -0,0 +1,24 @@
+package teammates.test.driver;
+
+import teammates.common.util.StringHelper;
+
+/**
+ * Holds additional methods for {@link StringHelper} used only in tests.
+ */
+
+public final class TestStringHelper {
+
+    private TestStringHelper() {
+        // utility class
+    }
+
+    /**
+     * Generates string of given length.
+     * @param length of which string to be generated
+     * @return generated Strings
+     */
```

Comment:
Oh I see 😅 Slight improvements suggested:
`Generates an arbitrary string of given length.`
`@param length of string to be be generated`
`@return generated string`